lib/fluent/plugin/in_tail.rb

```diff
@@ -244,8 +247,9 @@ def close_watcher(tw, close_io = true)
     end
 
     def close_watcher_after_rotate_wait(tw)
-      closer = TailWatcher::Closer.new(@rotate_wait, tw, log, &method(:close_watcher))
-      closer.attach(@loop)
+      timer_execute(:in_tail_close_watcher, @rotate_wait, repeat: false) do
```

Comment:
Executing event is only once but don't call `detach`.
So a watcher is still attached to event loop. It is resource leak.